pymongo/asynchronous/encryption_options.py

```diff
@@ -231,20 +231,20 @@ def __init__(
 
 
 class RangeOpts:
-    """Options to configure encrypted queries using the rangePreview algorithm."""
+    """Options to configure encrypted queries using the range algorithm."""
 
     def __init__(
         self,
         sparsity: int,
+        trim_factor: int,
```

Comment:
Making this a required argument is a breaking change. Is that intended? Are users expected to always specifiy trim_factor?

Reply:
Yeah, as per the [spec](https://github.com/mongodb/specifications/blob/59844221ade468044ec7b1e86d9db81eb7c6afad/source/client-side-encryption/client-side-encryption.md?plain=1#L1170), `trim_factor` is a required argument.

Reply:
I'm not sure that is the case. Adrian says:

> My understanding is that [trimFactor is] a knob for our tech support to debug performance issues and that average users should not touch it.